pacehrh/R/pace_coverage_config.R

```diff
@@ -0,0 +1,78 @@
+#' Load CoverageRates Values
+#'
+#' Read the annual task coverage values from the model inputs Excel file. The
+#' name and location of the model inputs Excel file is loaded from the global
+#' configuration JSON file.
+#'
+#' @param sheetName Sheet name from the model input Excel file
+#'
+#' @return Data table of per-year task coverage levels
+#'
+loadCoverageRates <- function(sheetName = .defaultCoverageRatesSheet){
+  traceMessage(paste0("Loading task coverage rates sheet ", sheetName))
+  
+  coverageRatesData <- NULL
+  
+  coverageRatesData <- loadTable(sheet = sheetName) # loadTable does validation
+  
+  hr <- colnames(coverageRatesData)
+  
+  # Sanity check: first two columns should be indicator and common name:
+  if (!identical(hr[1:2], c("Indicator", "CommonName"))) {
+    traceMessage(
+      paste0(
+        "Incorrect coverage rate config header names: (",
+        as.character(hr),
+        ")"
+      )
+    )
+    return(NULL)
+  }
+  
+  
+  # Total number of years in simulation
+  simYears <- length(BVE$years) 
+  
+  # All cols in coverageRatesData minus the indicator and common name cols
+  coverageYears <- length(coverageRatesData)-2 
+  
+  # Extend table to include all years if sim is longer than provided columns
+  if (simYears > coverageYears) {
```

Comment:
user should get an error instead of continue to silently assuming the coverage here